### PR TITLE
Regen for Gdk-Pixbuf v2.36.8

### PR DESCRIPTION
**Pull request by EPashkin**
Apply [last gir-files update](https://github.com/gtk-rs/gir-files/pull/14)

cc @GuillaumeGomez , @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
👍

File: src/auto/pixbuf.rs [hunk 1]
```diff
@@ -103,6 +103,13 @@ impl Pixbuf {
         }
     }
 
+    #[cfg(any(feature = "v2_36_8", feature = "dox"))]
```
Comment (GuillaumeGomez) [inline src/auto/pixbuf.rs]: This version isn't in the features' list.

**Comment by EPashkin** [comment]
Thanks, I missed this
Strange that `rust` don't nags

**Comment by GuillaumeGomez** [comment]
That's a cargo thing, not a rust one. 😉

**Comment by GuillaumeGomez** [comment]
All good, just waiting for CIs!

**Comment by EPashkin** [comment]
You right

**Comment by EPashkin** [comment]
Ping

**Comment by GuillaumeGomez** [comment]
Sorry, forgot about it!

**Pull request merged**